File: monitoreo.R
#! /usr/bin/env Rscript

args <- commandArgs(trailingOnly = TRUE)

usuario <- args[1]
ip <- args[2]
destino <- paste(usuario,"@",ip,":~/Crecimiento/Fotos/", sep = "")
archivos <- length(dir("/data/data/com.termux/files/home/storage/dcim/Camera"))

while (archivos == 1){
  nombre <- dir("/data/data/com.termux/files/home/storage/dcim/Camera")
  ruta1 <- paste("/data/data/com.termux/files/home/storage/dcim/Camera", nombre, sep = "/")
  envio <- paste("#! /usr/bin/bash \nscp", ruta1, destino, sep = " ")
  system("echo '#! /usr/bin/bash' > prueba.sh")
  writeLines(envio, con = "prueba.sh", sep = "\n")
  system("chmod +x prueba.sh")
  system("./prueba.sh")
  system("rm prueba.sh")
  borrar <- paste("rm", ruta1, sep = " ")
  system(borrar)
  #system("./movido.sh")
  Sys.sleep(as.numeric(args[3]))
  archivos <- length(dir("/data/data/com.termux/files/home/storage/dcim/Camera"))
}

File: telefono/monitoreo.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-
"""
Created on Mon Jan 11 17:59:14 2021

@author: Juan
"""

import os, sys
from os import remove

usuario = "USUARIO"
ip = "IP".strip()

destino = usuario + "@" + ip + ":~/Crecimiento/Fotos/"

nombre = os.listdir("/data/data/com.termux/files/home/storage/dcim/Crecimiento/")
ruta = "/data/data/com.termux/files/home/storage/dcim/Crecimiento/" + nombre[0]
envio = "#!/usr/bin/bash \nscp" + " " + ruta + " " + destino
f = open("envio.sh", "w")
f.write(envio)
f.close()
os.popen("chmod +x envio.sh")
os.popen("./envio.sh")
os.popen("rm envio.sh")
remove(ruta)
